README.md

```diff
@@ -381,10 +372,26 @@ A list of CSS classes to be added to the corresponding field. If you provide a f
 
 * `defaultValue(*)`
 Define the default value of the field in the creation form.
+### `choice` and `choices` Field Settings
+
+* `choices([{value: '', label: ''}, ...])`
+Define array of choices for `choice` type. A choice has both a value and a label.
+
+### `date` and `datetime` Field Settings
+
+* `format(string ['yyyy-MM-dd' by default])`
+Define the format for `date` and `datetime` types.
+
+* `parse(function [no change by default])`
```

Comment:
For the `date` field, there is a default parser to remove timezone, hours and minutes.